frontend/src/Taker.tsx

```diff
@@ -93,81 +108,98 @@ export default function App() {
         },
     });
 
+    const runningStates = ["Request sent", "Requested", "Contract Setup", "Pending Open"];
```

Comment:
Creating a mapping / state handling within the UI just makes our life harder and I would do it. 
We might as well just display what the daemon sends us (and do the mapping there, so we can enforce the mapping through rust code).

We already map the internal state of the application in `to_sse_event.rs`. I would just adapt the mapping to whatever we need and then the UI will just display what is sent. We might want to extend this mapping to separate label from state, so we can change the label easily while the underlying state does not have to change.

Reply:
Definitely worth to reconsider this. My goal was only on creating a _nicer_ ui.